.gitignore

```diff
@@ -5,3 +5,4 @@
 *~
 *.iml
 composer.phar
+app-prod.yaml
```

Comment:
No precedent, but I think it's better to say:
```
Copy the app.yaml to app-prod.yaml and edit it.
```

Rather than:
```
Edit the app.yaml
```
Because with the latter approach, you can accidentally commit your secret to your git clone.

I can move the `app.yaml` to `app.yaml.template` and do the opposite like:

```
Copy app.yaml.template to app.yaml and edit it.
```

What do you think?